src/components/Toaster/Toast/Toast.tsx

```diff
@@ -118,7 +118,9 @@ export const Toast = React.forwardRef<HTMLDivElement, ToastUnitedProps>(function
                         <Icon data={Xmark} />
                     </Button>
                 )}
-                {Boolean(content) && <div className={b('content')}>{content}</div>}
+                {Boolean(content) && (
+                    <div className={b('content', {'without-title': !title})}>{content}</div>
```

Comment:
Move title check to single var `hasTitle` please

```ts
const hasTitle = Boolean(title);
```

Reply:
Should I do the same with `Boolean(content)`?

Reply:
What alternative you thinking of?

Reply:
This is used only once, this is not required i think, but ok